src/core/openthread-core-default-config.h

```diff
@@ -159,6 +159,16 @@
 #define OPENTHREAD_CONFIG_MAX_TX_ATTEMPTS_INDIRECT_POLLS 4
 #endif
 
+/**
+ * @def OPENTHREAD_CONFIG_MAX_TRANSMIT_NUM_BROADCAST
+ *
+ * Maximum number of MAC layer transmission for a broadcast frame.
+ *
+ */
+#ifndef OPENTHREAD_CONFIG_MAX_TRANSMIT_NUM_BROADCAST
```

Comment:
This specifies how many times the MAC layer will transmit every time, not some number between 0 and MAX.  How about calling this something like `OPENTHREAD_CONFIG_TX_NUM_BCAST`.

Reply:
Done